megatron/model/fused_layer_norm.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         normalized_shape,
         eps=1e-5,
         no_persist_layer_norm=True,
-        sequence_parallel=False,
```

Comment:
@bclyang -- Shouldn't these remain `sequence_parallel` to match our previous support for megatron-style sequence parallelism (essentially just TP applied to layernorm and dropout)?